CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Fixed
 - Fixed `crate_authors!` macro usage, this macro can't return static string in new clap version. (#370)
 - Fixed mistake in description of the height getter in the `ServiceContext`. (#366)
+- Fixed #15 consensus on the threshold of 1/3 sleeping validators. (#388)
```

Comment:
I like the idea of having reference to extended description (such as issue), but I am not sure it will work with Jira.

Reply:
This is github issue

Reply:
I know, this isn't a problem right now, but perhaps we need some solution for future.

Reply:
I prefer to link only github issues, that would be for outside users.